mitmproxy/utils/debug.py

```diff
@@ -11,6 +11,11 @@
 from mitmproxy import version
 
 
+def remote_debug(host='localhost', port=4444):
+    import remote_pdb
+    remote_pdb.RemotePdb(host, port).set_trace()
+
+
```

Comment:
Leftover :)

Reply:
Not really. I added this as a facilitator for me to debug things. I thought that the `debug.py` file would be the place to have that. Is it a problem? If so, I can remove it.

Reply:
Only thing I forgot is that for this I need to add remote_pdb as a dependency for mitmproxy. Not sure if you are cool with that or not.

Reply:
let's put it in https://github.com/mitmproxy/mitmproxy/blob/master/examples/complex/remote_debug.py.